delivery_carrier_category/security/delivery_carrier_category.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
csv file seems the norm in Odoo and OCA modules for the ir.model.access records. (more synthetic, easy to edit).
(Just a remark / not a blocking point).

Reply:
@legalsylvain Tastes and colors... :smiley: I'd prefer this. Don't like inline editing, here columns are explicitly written (and I think it's not explicit in OCA as it let dev free to choose one or another).